modules/node_modules/@ncigdc/components/SurvivalPlotWrapper.js

```diff
@@ -156,7 +156,7 @@ const SurvivalPlotWrapper = ({
             {
               <div className="p-value">
                 <div style={styles.pValue}>
-                  {pValue && `Log-Rank Test P-Value = ${pValue.toExponential(2)}`}
+                  {`Log-Rank Test P-Value = ${pValue.toExponential(2)}`}
```

Comment:
this happens 
```
Uncaught TypeError: Cannot read property 'toExponential' of undefined
    at SurvivalPlotWrapper (SurvivalPlotWrapper.js:159)
```
maybe `(pValue || 0). toExponential(2)`?

(also ticket number in branch name & commit msg don't match)